core/ecschema-editing/src/Validation/SchemaCompareVisitor.ts

```diff
@@ -20,21 +21,28 @@ import { ISchemaComparer, SchemaComparerOptions } from "./SchemaComparer";
 export class SchemaCompareVisitor implements ISchemaPartVisitor {
   private _schemaB: Schema;
   private _schemaComparer: ISchemaComparer;
-  private _lookupFullName: boolean;
 
   /**
    * Initializes a new SchemaCompareVisitor instance.
    * @param schemaComparer The [[SchemaComparer]] to use to compare each item of the schema.
    * @param schemaToCompare The second or 'B' schema to compare against the schema being traversed (Schema A).
    */
-  constructor(schemaComparer: ISchemaComparer, schemaToCompare: Schema, options?: SchemaComparerOptions) {
+  constructor(schemaComparer: ISchemaComparer, schemaToCompare: Schema) {
     this._schemaComparer = schemaComparer;
     this._schemaB = schemaToCompare;
-    this._lookupFullName = options?.compareItemFullName ?? true;
   }
 
+  /**
+   * Called to look up higher level schema items.
+   * @param item
+   * @returns
+   */
   private async lookupItem<T extends SchemaItem = SchemaItem>(item: SchemaItem) {
```

Comment:
I know you did not name this method but I suggest changing it to something like `lookupItemInSchemaB` or something.  Having the same name as the schema method is confusing and in fact it looks like one case was not updated to call this method in this file.

Reply:
Changed it to `lookupItemInSchemaB` and updated the case that was not calling this function.